plugin/src/main/kotlin/com/supercilex/gradle/versions/internal/Validation.kt

```diff
@@ -20,11 +23,15 @@ internal fun validateRuntime() {
         """.trimMargin()
     }
 
-    check(agpVersion >= MIN_AGP_VERSION) {
-        """
-        |Version Orchestrator's minimum Android Gradle Plugin version is at least
-        |$MIN_AGP_VERSION and yours is $agpVersion. Find the latest version and upgrade
-        |instructions at https://developer.android.com/studio/releases/gradle-plugin.
-        """.trimMargin()
+    project.plugins.withType<AppPlugin> {
+        val agpVersion = project.extensions.findByType<ApplicationAndroidComponentsExtension>()?.pluginVersion
```

Comment:
In AGP 7.0 getting the version is not too straightforward, is this method acceptable?

Reply:
Yup!